docs/marketing/detailed-description.md

```diff
@@ -1,8 +1,10 @@
+Built for developers, [**KOMOJU**](https://komoju.com) is the easiest way to integrate Japanese payments into your Magento store.
```

Comment:
All the changes in this file were made according to the [Magento requirements](https://devdocs.magento.com/marketplace/sellers/product-descriptions.html), mainly having the first paragraph be a description of the company, and having a `Security` section to explain how the payment process will occur.